docs/scripts/uploadTestFiles.py

```diff
@@ -49,36 +58,52 @@ def create_folder_structure(
             root/subdir2/file1.txt
             root/subdir2/file2.txt
 
-    :param path: _description_
-    :param depth_of_directory_tree: _description_
-    :param num_sub_directories: _description_
-    :param num_files_per_directory: _description_
-    :param total_size_of_files_mbytes: _description_
-    :return: _description_
+    Arguments:
+        path: The path to the root directory
+        depth_of_directory_tree: The depth of the directory tree
+        num_sub_directories: The number of subdirectories to create
+        num_files_per_directory: The number of files to create in each directory
+        total_size_of_files_mib: The total size of all files in MiB
+
+    Returns:
+        The total number of directories, total number of files, and the size of each
+        file in bytes
     """
     # Calculate total number of files and size of each file
     total_dirs = sum(
         [num_sub_directories**i for i in range(1, depth_of_directory_tree + 1)]
     )
     total_files = total_dirs * num_files_per_directory
-    total_size_of_files_bytes = total_size_of_files_mbytes * 1024 * 1024
+    total_size_of_files_bytes = total_size_of_files_mib * MiB
```

Comment:
Why is this being converted to mib, if the argument should already be fed in mib?